src/Swarm/TUI/Controller.hs

```diff
@@ -96,14 +99,20 @@ handleEvent s (VtyEvent (V.EvKey (V.KChar '\t') [])) = continue $ s & uiState .
 handleEvent s (VtyEvent (V.EvKey V.KBackTab []))     = continue $ s & uiState . uiFocusRing %~ focusPrev
 handleEvent s (VtyEvent (V.EvKey V.KEsc []))
   | isJust (s ^. uiState . uiError) = continue $ s & uiState . uiError .~ Nothing
-handleEvent s (VtyEvent (V.EvKey (V.KChar 'q') [V.MCtrl])) = shutdown s
+handleEvent s (ControlEvent 'q') = shutdown s
+handleEvent s (ControlEvent 'g') = setFocus s InfoPanel
+handleEvent s (ControlEvent 'r') = setFocus s REPLPanel
+handleEvent s (ControlEvent 'w') = setFocus s WorldPanel
```

Comment:
I am starting to wonder whether it is a mistake to bind Ctrl-Q to quit and Ctrl-W to something else.  You just know someone is going to accidentally quit when they meant to switch to the world panel.  At the very least we should add a confirmation dialog when the user quits (which is probably a good idea anyway).

I don't know, maybe rather than trying to come up with mnemonic keys for the panels, we should just bind them to Ctrl-J,K,L or something like that?

Reply:
Good point, I first tried `ctrl-m` for map, but this is not working for the same reason as `ctrl-i`.

The Info panel could also be mapped to `ctrl-e`, as `e` is often used for inventory in such game.

Another solution would be to use F1/F2/... keys, which might be easier to use without ctrl modifier.

Reply:
Ah, yeah, I think I like the idea of using Fn keys.

Reply:
Alright I'll update the match for F keys.

Reply:
It's not so easy to type function keys on all keyboards, and even when it is leaving the home row kinda sucks. What about just assigning to, say, Ctrl-1, Ctrl-2 etc? Or Alt-anything?

Reply:
That's true.  Well, Ctrl-1 has the same problem as Ctrl-W: it is right next to Ctrl-Q.  We could also try Alt but that is a bit tricky too as far as I understand it, because what ends up being called "Alt" varies a lot depending on your keyboard and setup etc.

Reply:
Of course the real solution is to let the user assign their own keyboard shortcuts, but that sounds like too much work right now. =)

Reply:
The latest commit use the Meta modifiers with `e` for the info, `m` for the world and `r` for the repl. I think using Meta is common for such shortcuts, and it seems to match the left alt key.